docs/howtos/redistimeseries/using-dotnet/index.md

```diff
@@ -0,0 +1,136 @@
+---
+id: index-usingdotnet
+title: Processing Time Series data with Redis and .NET
+sidebar_label: RedisTimeSeries and With .NET
+slug: /howtos/redistimeseries/using-dotnet
+authors: [steve]
+---
+
+Time Series data can be used to measure anything from remote sensor readings to stock market feeds. Using time-series in .NET is a snap with Redis and [NRedisTimeSeries](https://github.com/RedisTimeSeries/NRedisTimeSeries), and in this tutorial, we'll explore how we can use them.
+
+## Create your Project
+
+Start out by creating a project with the command:
+
+```bash
+dotnet new console -n TimeSeriesDemoApp
+```
+
+Next, inside the `TimeSeriesDemoApp` directory, run the command: 
+
+```bash
+dotnet add package NRedisTimeSeries
+```
+
+## Get a RedisTimeSeries Database
+
+The next step is to get a RedisTimeSeries database up and running. The easiest way to do that for development purposes is to use docker:
+
+```
+docker run -p 6379:63379 redislabs/redistimeseries
+```
+
+If you are well past getting started and want to get something into your production, your best bet is to run it in [Redis Enterprise](http://localhost:3000/howtos/redistimeseries/getting-started).
+
+## Connecting to Redis
+
+Open the `Program.cs` file, in here, create a new ConnectionMultiplexer using a connection string (which will vary based on what deployment you're using). Then, for our basic docker setup, you'll just run:
+
+```csharp
+var muxer = ConnectionMultiplexer.Connect("localhost");
+var db = muxer.GetDatabase();
+```
+
+## Create a TimeSeries
+
+Now that you've gotten a handle to Redis, your next step is to initialize a TimeSeries. This will be a bit of a toy example. We are going to start off by just creating a Time Series called `sensor`, we will set its retention period to 1 minute, and we just give it an `id` label of `sensor-1`:
+
+```csharp
+await db.TimeSeriesCreateAsync("sensor", 60000, new List<TimeSeriesLabel>{new TimeSeriesLabel("id", "sensor-1")});
+```
+
+## Producer Task
+
+Next, we'll create a task that will run a consumer in the background. Every second it will send a random integer between 1 and 50 into our time series a the current time.
```

Comment:
```suggestion
Next, we'll create a task that will run a consumer in the background. Every second it will send a random integer between 1 and 50 into our time series.
```